benchmarks/source-sanity/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "gatsby-starter-hello-world",
+  "private": true,
+  "description": "A simplified bare-bones starter for Gatsby",
+  "version": "0.1.0",
+  "license": "MIT",
+  "scripts": {
+    "build": "gatsby build",
+    "build:send": "cross-env BENCHMARK_REPORTING_URL=true gatsby build",
+    "develop": "gatsby develop",
+    "format": "prettier --write \"**/*.{js,jsx,json,md}\"",
+    "start": "npm run develop",
+    "serve": "gatsby serve",
+    "update-post": "NODE_ENV=development node update-post.js"
```

Comment:
```suggestion
    "data-update": "NODE_ENV=development node update-post.js"
```

Reply:
Also should probably be production, although I don't see a reason this would break anything 🤷

Reply:
This does actually need to be `production`.